src/TriArithmetic/conversions.js

```diff
@@ -21,3 +21,36 @@ export function wordToDec(word) {
   }
   return dec;
 }
+
+/**
+ * Converts a dec to balanced ternary representation
+ * @param {decimal} decimal
+ * @returns {balancedTernary} - ternary value
+ */
+export function decimalToBalancedTernary(decimal) {
+  function decimalToBalancedTernaryDigit(digit) {
+    if (digit === 0) return 0;
+    if (digit === 1) return 1;
+    if (digit === 2) return -1;
+  }
+
+  // Check if the decimal number is within the bounds
+  const maxDecimal = 3 ** (WORD_SIZE - 1) - 1;
```

Comment:
This isn't quite right. For example, with 2 digits I can represent maximally "11" = (1 * 3 ** 0) + (1 * 3 ** 1) = 4, but 3 ** (2 - 1) - 1 = 2